src/commands/build/wranglerjs/bundle.rs

```diff
@@ -109,31 +112,20 @@ pub fn create_prologue() -> String {
 }
 
 // This metadata describe the bindings on the Worker.
-fn create_metadata(bundle: &Bundle) -> String {
-    info!("create metadata; wasm={}", bundle.has_wasm());
+fn create_metadata(bundle: &Bundle) -> Result<String, serde_json::error::Error> {
+    let mut bindings = vec![];
+
     if bundle.has_wasm() {
-        format!(
-            r#"
-                {{
-                    "body_part": "script",
-                    "bindings": [{{
-                        "name": "{name}",
-                        "type": "wasm_module",
-                        "part": "{name}"
-                    }}]
-                }}
-            "#,
-            name = bundle.get_wasm_binding(),
-        )
-        .to_string()
-    } else {
-        r#"
-            {
-                "body_part": "script"
-            }
-        "#
-        .to_string()
+        bindings.push(Binding::new_wasm_module(
+            bundle.get_wasm_binding(),
+            bundle.get_wasm_binding(),
```

Comment:
twice?

Reply:
Once is the name and the other the multipart part.